src/Silex/Provider/SessionServiceProvider.php

```diff
@@ -50,6 +50,8 @@ public function register(Application $app)
             return new Session($app['session.storage']);
         });
 
+        $app->configure('session', ['isGlobal' => true]);
```

Comment:
Why do we need this?

Reply:
That's Rene's decision, the issue is that global services can only be extended by global service 😞, this is the solution.